index.js

```diff
@@ -43,13 +43,14 @@ module.exports = fp(/**
                     if (!build) {
                         await buildPromise;
                     }
-                    return nuxt.render({...request.raw, ...request}, reply.res);
+                    reply.sent = true;
+                    return nuxt.render(request.raw, reply.res);
```

Comment:
I used object destruction to make methods/objects attached by fastify plugins available in the nuxt context. As example sessions.

Reply:
webpack-hmr requires first argument of `nuxt.render` to be an instance of `http.IncomingMessage`, so it will cause error if two objects are simply concatenated.

example:
```
function Foo () {}
var a = new Foo()
var b = {...a, ...{}}
a instanceof Foo // true
b instanceof Foo // false
```

So, to make fastify-original request params available in nuxt, additional hacks are needed like
```
let im = {...request.raw, ...request};
im.__proto__ = IncomingMessage.prototype;
return nuxt.render(im, reply.res);
```
but it won't work and I don't think it good manners.